server/src/models/job_applications.model.js

```diff
@@ -0,0 +1,94 @@
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../configs/sequelize");
+const JobPosting = require("./job_posts.model");
+
+const JobApplication = sequelize.define(
+  "job_applications",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    job_posting_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: JobPosting,
+        key: "id",
+      },
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        isEmail: true,
+      },
+    },
+    phone: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        is: /^\d{10}$/,
+      },
+    },
+    postal_code: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        is: /^[A-Za-z]\d[A-Za-z][ -]?\d[A-Za-z]\d$/,
+      },
+    },
+    resume: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    status_in_canada: {
+      type: DataTypes.ENUM(
+        "Citizen",
+        "PR",
+        "conventional refugee",
+        "student visa",
```

Comment:
The first letters of each of these options should be capitalized, similar to how capitalization is done for application_status